site/multiembedder/MultiEmbedder.tsx

```diff
@@ -202,7 +201,9 @@ class MultiEmbedder {
         } else {
             figure.classList.remove(GRAPHER_PREVIEW_CLASS)
 
-            const grapherPageConfig = deserializeJSONFromHTML(html)
+            const grapherPageConfig = await fetch(
+                `${fullUrl}.config.json`
```

Comment:
This will not work if `fullUrl` contains query params!

Reply:
Thanks! Fixed this.